Source/WarpX.cpp

```diff
@@ -186,50 +187,54 @@ amrex::IntVect m_rho_nodal_flag;
 
 WarpX* WarpX::m_instance = nullptr;
 
-namespace
-{
-
-    [[nodiscard]] bool
-    isAnyBoundaryPML(
-        const amrex::Array<FieldBoundaryType,AMREX_SPACEDIM>& field_boundary_lo,
-        const amrex::Array<FieldBoundaryType,AMREX_SPACEDIM>& field_boundary_hi)
-    {
-        constexpr auto is_pml = [](const FieldBoundaryType fbt) {return (fbt == FieldBoundaryType::PML);};
-        const auto is_any_pml =
-            std::any_of(field_boundary_lo.begin(), field_boundary_lo.end(), is_pml) ||
-            std::any_of(field_boundary_hi.begin(), field_boundary_hi.end(), is_pml);
-        return is_any_pml;
-    }
+namespace {
+
+[[nodiscard]] bool
```

Comment:
I know that it is not recommended, but I actually like the indentation inside a namespace. We may want to discuss this.

Reply:
I would agree that it is better to use indentation inside a name space. This makes it readily clear that the routines defined are in a defined space.

Reply:
I think it should be possible to guarantee this by replacing
```
NamespaceIndentation: None
```
with 
```
NamespaceIndentation: All
```